### DISCUSSION TITLE
byo grafana - recommended method to import linkerd dashbaords

### DISCUSSION THREAD

**Discussion by naseemkullah** [linkerd/linkerd2#5364] [Ideas]
Consolidating linkerd dashboards into a byo grafana (byog) is great UX since folk  have one less url to remember for all their data viz needs...however getting linkerd dashboards into a byog and keeping them in sync with upstream linkerd dashboards is desired. What would be the best approach?

@Pothulapati mentioned these approaches:

- https://github.com/andrew-waters/linkerd2-mixin
- https://medium.com/@chris_linguine/adding-linkerd-2-x-data-to-your-main-grafana-dashboard-4371cc1fa7e3

mixin approach might be nice? is there a way to integrate Andrew Waters' work into the linkerd2 repo?

The second approach is nice in the sense that it is straightforward but manual and not viable to keep doing over and over to stay in sync with upstream dashboards.

**Comment by adleong** [comment]
This looks neat!  If you try either of these two approaches I'd be very interested to see how it looks.
